subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/BuildVerdictPluginExtension.kt

```diff
@@ -35,11 +36,11 @@ abstract class BuildVerdictPluginExtension(
         )
     }
 
-    fun onTaskFailure(name: String, producer: (Task) -> String) {
+    fun onTaskFailure(name: String, producer: (Task) -> SpannedString) {
```

Comment:
This PR is incompatible with the old API and will break current users because I don't see a reason to make this change in two steps

Reply:
Some more context to this (as i went to our project to find current clients):

```
    onTaskFailure("detektAll") {
        File(detekt.reportsDir, "detekt.txt").readText()
    }
```

So this lambda will be red because of changed return value;

look ok to me, minor thing to remember on version update

Reply:
see: https://github.com/avito-tech/avito-android/pull/696#discussion_r549892299

Reply:
Yes, I want to break compatibility and fix it in the release commit